htop.c

```diff
@@ -162,6 +167,24 @@ static void millisleep(unsigned long millisec) {
    }
 }
 
+static void record_btime() {
```

Comment:
This operation can be done in `linux/LinuxProcessList.c`, in the initialization function of that file.